bin/start

```diff
@@ -1,22 +1,36 @@
 #!/usr/bin/env bash
 
+if [ -f .env ]; then
+  export $(echo $(cat .env | sed 's/#.*//g'| xargs) | envsubst)
```

Comment:
Is this required?
 I don't think it is required for production workloads, the container itself will already have all ENV VARs on the `.env` file parsed and loaded by the composer.
This IMO will only add more delay for the start of the service,

Reply:
Yep - just forgot to remove it